docs/adr/0015-use-css-variables-and-scss.md

```diff
@@ -0,0 +1,181 @@
+# Use CSS Variables, Prefer SCSS over CSS-in-JS
+
+- Deciders: Lauren Zugai, Jody Heavener
+- Date: 2020-04-27
+
+Technical Story: [Github issue](https://github.com/mozilla/fxa/issues/4808)
+
+## Context and Problem Statement
+
+The [Settings Redesign](https://github.com/mozilla/fxa/issues/3740) project offers an ideal opportunity to review CSS conventions and architecture for FxA to use not only while building new components for this project but also for FxA moving forward.
+
+This [part 1](https://github.com/mozilla/fxa/issues/4808) of [2](https://github.com/mozilla/fxa/issues/5087) ADR concerns the use of CSS variables, how they come into play with SCSS (vs using SCSS variables), CSS-in-JS considerations, and CSS modules.
+
+Note that 1) "CSS variables" are technically called "CSS custom properties" but will be referred to as the common "variable" nomenclature throughout this ADR and 2) SASS is a CSS preprocessor while SCSS is our preferred syntax of SASS.
+
+## Decision Drivers
+
+- Simplicity and clarity
+- DRY and reusable styles
+- Easy integration and use
+- Current engineering experience and preferences
+
+## Considered Options
+
+### Styling variables
+
+- SASS variables
+- CSS variables
+- A combination of SASS and CSS variables
+
+### Styling placement
+
+- CSS-in-JS with a helper lib like [`styled-components`](https://styled-components.com/) or [`react-jss`](https://cssinjs.org/react-jss/)
+- CSS modules
+- SCSS
+
+## Decision Outcome
+
+Chosen options: "CSS variables" and "SCSS", because:
+
+- CSS variables are native to the browser; they have good browser support and can be used and set without a preprocessor and after preprocessor compilation. They are targetable by JavaScript, easily themed, and can be scoped globally or on an element, providing options for usage.
+- CSS variables can be used in SCSS without impeding core functionality SASS offers like nested selectors plus functions and mixins that don't rely on variable values.
+- Scoped component solutions (CSS-in-JS, CSS modules) encourage self-contained components rather than building the UI as a whole. While FxA will have some component reuse, FxA will likely reap more benefits from a class based approach with shared styles as a much DRYer and consistent approach.
```

Comment:
Good call out. This also reminds me that by using classes, it may actually make some sharing cleaner because we won't have a bunch of if/elses in the styles based on props. Will update.